src/main.rs

```diff
@@ -43,13 +43,12 @@ async fn main() -> std::io::Result<()> {
         App::new()
             .wrap(Logger::default())
             .wrap(Logger::new("%a %{User-Agent}i"))
-            .app_data(handlebars_ref.clone())
-            .service(fs::Files::new("/static", "./static").show_files_listing())
             .service(routes::index_get)
-            .service(routes::search_post)
-            .service(routes::search_get)
-            .service(routes::mod_page_get)
-            .service(routes::mod_create_get)
+            .service(routes::mod_search)
+            //.service(routes::search_get)
+            //.service(routes::mod_page_get)
+            //.service(routes::mod_create_get)
+            .default_service(web::get().to(routes::not_found))
     })
     .bind("127.0.0.1:8000")?
```

Comment:
Since we're adding `dotenv` config, this seems like something that should be there as well.